apps/src/blockly/addons/unknownBlock.js

```diff
@@ -0,0 +1,9 @@
+export const UNKNOWN_BLOCK = {
+  unknownBlock: true,
+  init: function() {
+    this.setHSV(0, 0, 0.8);
+    this.appendDummyInput().appendField('unknown block', 'NAME');
```

Comment:
Does "unknown block" get localized when the user sees this block?

Reply:
This block should never be seen by external users- it indicates that something is broken in the level. The only reason to show this block instead of throwing an error is to make it easier for curriculum writers to debug problems.